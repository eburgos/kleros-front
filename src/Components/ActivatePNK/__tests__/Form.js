import React from 'react'
import { Form } from '../Form'

describe('Form Exists', () => {
  const component = shallow(<Form />)

  test('should render component properly', () => {
    expect(component).toMatchSnapshot()
  })

  test('should render Field component properly', () => {
    expect(component.find('Field').length).toBe(1)
  })

  test('should render button text', () => {
    expect(component.find('button').text()).toBe('Activate')
  })
})
