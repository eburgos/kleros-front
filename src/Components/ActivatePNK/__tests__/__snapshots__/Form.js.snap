// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form Exists should render component properly 1`] = `
<form
  className="Form-container"
>
  <div
    className="amount-container"
  >
    <Field
      component={[Function]}
      defaultValue={0}
      innerClassName="input-number"
      name="amount"
      required={true}
      step={1e-17}
      type="number"
    />
  </div>
  <div
    className="button-container"
  >
    <button
      className="submit"
      type="submit"
    >
      Activate
    </button>
  </div>
</form>
`;
